repositories/BUILD

```diff
@@ -15,3 +15,22 @@
 licenses(["notice"])  # Apache 2.0
 
 package(default_visibility = ["//visibility:public"])
+
+load("@bazel_skylib//:skylark_library.bzl", "skylark_library")
```

Comment:
https://github.com/bazelbuild/bazel-skylib/blob/master/skylark_library.bzl#L1

Reply:
yes, its being deprecated, but the latest release (0.5.0) does not include this change yet, so will do that in upcoming PR once the latest release includes it.

Reply:
Ack.